en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/configure-logs-enforcer.md

```diff
@@ -0,0 +1,169 @@
+|-------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| OFF   | The highest possible log level. This is intended for disabling logging.                                                                                                                                                                                                         |
+| FATAL | Indicates server errors that cause premature termination. These logs are expected to be immediately visible on the command line that you used for starting the server.                                                                                                          |
+| ERROR | Indicates other runtime errors or unexpected conditions. These logs are expected to be immediately visible on the command line that you used for starting the server.                                                                                                           |
+| WARN  | Indicates the use of deprecated APIs, poor use of API, possible errors, and other runtime situations that are undesirable or unexpected but not necessarily wrong. These logs are expected to be immediately visible on the command line that you used for starting the server. |
+| INFO  | Indicates important runtime events, such as server startup/shutdown. These logs are expected to be immediately visible on the command line that you used for starting the server . It is recommended to keep these logs to a minimum.                                           |
+| DEBUG | Provides detailed information on the flow through the system. This information is expected to be written to logs only. Generally, most lines logged by your application should be written as DEBUG logs.                                                                        |
+| TRACE | Provides additional details on the behavior of events and services. This information is expected to be written to logs only.                                       |
+
+### Setting the Threshold
+
+Filters log entries based on their level. For example, threshold set to 'WARN' will allow the log entry to pass into appender. If its level is `WARN`, `ERROR` or `FATAL`, other entries will be discarded. This is the minimum log level at which you can log a message. See descriptions of the available log levels.
+
+!!! Note
+    The log level setup at `appender` might be overidden with the properties defined at logger. For Example, if we set the threshold at `appender` level as `DEBUG` and logger level as `ERROR`, it will only show the logs upto `ERROR` level.
```

Comment:
```suggestion
    The log level set up at `appender` might be overridden with the properties defined at logger. For example, if we set the threshold at the `appender` level as `DEBUG` and logger level as `ERROR`, it will only show the logs up to the `ERROR` level.
```